styles/main.css

```diff
@@ -53,6 +56,10 @@ ul {
     text-indent: 3em;
 }
 
+.favourites {
+    clear: both;
+}
+

```

Comment:
This rule makes it so the second section always clears the floated mugshot element